modules/EnsEMBL/Web/Component/Variation/PopulationGraphs.pm

```diff
@@ -162,13 +164,13 @@ sub content {
     my $show     = $hub->get_cookie_value("toggle_population_freq_$sp") eq 'open';
     
     $html .= sprintf('
-      <div class="population_freq_%s population_genetics_pie">
+      <div class="population_freq_%s population_genetics_pie" id="population_freq_%s">
         <div class="toggleable" %s>
           <div><p><b>%s sub-populations</b></p></div>
           %s
         </div>
       </div>
-    ', $sp, $show ? '' : 'style="display:none"', $sp, $sub_html);
+    ', $sp, $sp, $show ? '' : 'style="display:none"', $sp, $sub_html);
```

Comment:
Hi Laurent
Looks like you have an extra $sp in here, is that ok? or is the diff which is confusing.

Ridwan